cmd/flags/get.go

```diff
@@ -57,11 +58,6 @@ func NewGetCmd(client flags.Client) (*cobra.Command, error) {
 
 func runGet(client flags.Client) func(*cobra.Command, []string) error {
 	return func(cmd *cobra.Command, args []string) error {
-		// rebind flags used in other subcommands
```

Comment:
Turns out we don't need to do this. It will be easier to remove these with the generated commands since they'll only be written once.